fsi/init.go

```diff
@@ -197,7 +197,7 @@ func (fsi *FSI) InitDataset(p InitParams) (name string, err error) {
 		}
 	}
 
-	if err = fsi.repo.Logbook().WriteDatasetInit(context.TODO(), ref.Name); err != nil {
+	if _, err = fsi.repo.Logbook().WriteDatasetInit(context.TODO(), ref.Name); err != nil {
```

Comment:
I'd like make sure we're returning this dropped InitID. Can we add a TODO saying `fsi.InitDataset` should return a `dsref.Ref` once it gets an `InitID` field, and set ref.InitID here?

Reply:
Will do. Avoided this at first because it still gets dropped, but having the TODO around is a good stop gap.